embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/arch/schema/EmbType.kt

```diff
@@ -57,6 +57,8 @@ internal sealed class EmbType(type: String, subtype: String?) : TelemetryType {
 
         internal object Exception : System("exception")
 
+        internal object InternalError : System("internal")
```

Comment:
why not a name that indicates that it's some sort of error or warning?

Reply:
This is what was suggested with Pablo/Austin